net-misc/pcapfix/pcapfix-1.1.4.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Repair and fix broken pcap files"
+HOMEPAGE="http://f00l.de/pcapfix/"
+SRC_URI="http://f00l.de/pcapfix/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug"
+
+RDEPEND=""
+DEPEND="${RDEPEND}"
```

Comment:
This is rather useless :) You can leave them empty or even remove these variables if nothing goes into them.